lib/Command/ExpirePassword.php

```diff
@@ -95,20 +113,38 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 			return 1;
 		}
 
-		$date = new \DateTime($input->getArgument('expiredate'));
-		$date->setTimezone(new \DateTimeZone('UTC'));
-		$value = $date->format('Y-m-d\TH:i:s\Z'); // ISO8601 with Zulu = UTC
+		$expireDate = new \DateTime();
+		$expireDate->setTimezone(new \DateTimeZone('UTC'));
+		$expireDate->setTimestamp($this->timeFactory->getTime());
+		$expireDate->modify($input->getArgument('expiredate'));
+
+		$oldDate = new \DateTime();
+		$oldDate->setTimezone(new \DateTimeZone('UTC'));
+		$oldDate->setTimestamp($this->timeFactory->getTime());
+		$oldDate->modify($input->getArgument('expiredate'));
```

Comment:
Someone could extract method to refactor these 2 blocks that make ``$expireDate`` and ``$oldDate``, or is there magic in PHP to really copy/clone and object to make ``$oldDate`` a real copy of ``$expireDate`` then modify ``$oldDate`` below.